### ISSUE TITLE
Issues while navigating list view in CRM

### ISSUE THREAD

**Issue by nehasan16** [frappe/crm#401]
When working on a list of leads, I faced the following issues which made my experience really poor. 

1. I open a deal, address it, and press "back"
<img width="1120" alt="Screenshot 2024-10-06 at 3 03 34 PM" src="https://github.com/user-attachments/assets/6b7be312-86fc-47ab-adf9-1f36b29e776b">

2. After pressing back, I lost track of which lead I had really opened and which was the next in line. :( 

Ideally, once I am back to the list view, the view before opening the deal should persist, and the deal I just opened should is clearly visible. In ERPNext this is achieved by making the name of then opportunity "unbolded". In email, this is achieved by giving a different color to unopened emails and a different color to opened emails. 

Basically, currently, it is very hard to go through a list of deals easily. The UX can be much better.

**Comment by NehaSan16** [comment]
@shariquerik Can we please check this?

**Comment by michellealva** [comment]
@nehasan16 the view is persistent, I just checked. However, the bold text one can be done. Can you confirm the view thing if it works for you?
